tests/src/integration/cloudant/manifest.yaml

```diff
@@ -0,0 +1,38 @@
+# This manifest file reads following env. variables:
+#   CLOUDANT_USERNAME
+#   CLOUDANT_PASSWORD
+
+package:
+    name: cloudant-sample
+    dependencies:
+        # binding cloudant package named openwhisk-cloudant
+        openwhisk-cloudant:
+            location: /whisk.system/cloudant
+            inputs:
+                username: $CLOUDANT_USERNAME
+                password: $CLOUDANT_PASSWORD
+                host: ${CLOUDANT_USERNAME}.cloudant.com
```

Comment:
@daisy-ycguo do we support this structure, i have tried many times and it does not replace env. variable $CLOUDANT_USERNAME in ${CLOUDANT_USERNAME}.cloudant.com.

Reply:
We don't support this. This manifest file doesn't work.

Reply:
@pritidesai daisy opened another issue here to address this as a new feature... issue https://github.com/apache/incubator-openwhisk-wskdeploy/issues/501